src/components/PageWithSkillList.tsx

```diff
@@ -253,7 +252,7 @@ const PageWithSkillList = ({
                     <Link href={`/skills/${context}/${category.name}`}>
                       <button
                         className={`${
-                          add && context != "zenika"
+                          add && context === "mine"
```

Comment:
```suggestion
                          add 
```

Could be removed as it is now checked line 245